aibolit/extract_method_baseline/clustering_test.py

```diff
@@ -0,0 +1,43 @@
+from collections import OrderedDict
```

Comment:
Format the test as a unit test in `test/extract_method_baseline/` folder.

Reply:
it is no a unittest